python/dolfinx/fem/linearproblem.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020 Jørgen S. Dokken
+#
+# This file is part of DOLFINX (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+import typing
+
+from petsc4py import PETSc
+from ufl import form
```

Comment:
Use module name for UFL function/classes, otherwise it get confusing with names like 'form'.

```
import ufl
foo = ufl.form()
```